db/migrate/20230630162646_add_user_attributes.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Add more attributes to User which can be fetched from authenticators
+class AddUserAttributes < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users, bulk: true do |t|
+      t.string :provider_username
+      t.string :first_name
+      t.string :last_name
+      t.string :phone_number
```

Comment:
**nitpick (blocking):** Can we please remove the `phone_number` field, I'm trying to keep IRIDA Next away from being a directory. One feature that we plan on adding in the future is ways for users to request access from other users in the system, which removes the need to provider other users phone numbers.

Reply:
Removed phone_number in https://github.com/phac-nml/irida-next/pull/134/commits/d489c2a65df8bad1b9a65273ff23e1eabafbc4c3